core/deployment/src/main/java/io/quarkus/deployment/pkg/ManifestConfig.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.deployment.pkg;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class ManifestConfig {
+
+    /**
+     * Manifest specification version.
+     */
+    @ConfigItem(defaultValue = "1.0")
+    public String version;
+    /**
+     * If the Implementation information should be included in the runner jar's MANIFEST.MF.
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean implementationEntries;
```

Comment:
Sorry to be picky but I think `addImplementationEntries` would be more explicit.